auth/server/src/main/java/com/seagates3/controller/AccountController.java

```diff
@@ -234,6 +243,25 @@ int getTotalCountOfAccounts() throws DataAccessException {
       return accounts.length;
     }
 
+    /**
+ * Generate account uidNumber and check if its unique in ldap
+ * @throws DataAccessException
+ */
+   private
+    String generateUniqueuidNumber() throws DataAccessException {
+      Account account;
+      String accountuidNumber;
+      for (int i = 0; i < 5; i++) {
+        accountuidNumber = KeyGenUtil.createAccountuidNumber();
```

Comment:
keep this for loop under if(!account.exist()).if it already exists no need of this loop to process ahead.